Wikipedia/Defines/WMFDeprecationMacros.h

```diff
@@ -0,0 +1,19 @@
+//
+//  WMFDeprecationMacros.h
+//  Wikipedia
+//
+//  Created by Brian Gerstle on 11/11/15.
+//  Copyright © 2015 Wikimedia Foundation. All rights reserved.
+//
+
+#ifndef WMFDeprecationMacros_h
+#define WMFDeprecationMacros_h
+
+#if __IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_9_0
+#define WMF_DEPRECATED_WHEN_DEPLOY_AT_LEAST_9 __deprecated_msg("This API was only necessary when deployment target was < 9. Please remove any code being executed when OS is less than 9.")
```

Comment:
I'm not sure about this… we were on a positive path to removing warnings but lately it is out of control again. The warning panel is all but useless at the point - with all useful warnings being drowned out by things like this. Something we should probably discus at code review. 

Seeing 500 warnings about things we should do someday when we have time isn't useful. Thats what engineering backlog tickets are for.
Seeing 5 warnings bout things I just changed and possibly broke is very useful and helps me prevent introducing bugs.